src/test/java/tests/ZipCodesTest.java

```diff
@@ -27,8 +30,11 @@ public void postZipCodesTest() {
     }
 
     @Test
+    @Step("POST Zipcode duplicate in list, Request payload: {payload}")
```

Comment:
The Step annotation should be added to the methods, not to the tests. Try to use it for Client classes (UserClient, ZipcodeClient, etc). As a rule, this annotation on the method adds all the parameters of the method to the report. Thus, there is no need to explicitly specify the payload. Try this and check the report